Mathlib/Analysis/Complex/UpperHalfPlane/Topology.lean

```diff
@@ -87,9 +87,9 @@ lemma subset_strip_of_isCompact {K : Set ℍ} (hK : IsCompact K) :
     ∃ A B : ℝ, 0 < B ∧ K ⊆ verticalStrip A B := by
   rcases K.eq_empty_or_nonempty with rfl | hne
   · exact ⟨1, 1, Real.zero_lt_one, empty_subset _⟩
-  obtain ⟨u, _, hu⟩ := hK.exists_isMinOn hne continuous_im.continuousOn
-  obtain ⟨v, _, hv⟩ := hK.exists_isMaxOn hne (_root_.continuous_abs.comp continuous_re).continuousOn
-  exact ⟨|re v|, im u, u.im_pos, fun k hk ↦ ⟨isMaxOn_iff.mp hv _ hk, isMinOn_iff.mp hu _ hk⟩⟩
+  obtain ⟨v, _, hv⟩ := hK.exists_isMinOn hne continuous_im.continuousOn
```

Comment:
If you care, you could also switch these two lines. (But this is really nano-comment territory.)